Card/src/main/java/com/paypal/android/card/CardRequestFactory.kt

```diff
@@ -30,18 +30,25 @@ internal class CardRequestFactory {
                 cardJSON.put("billing_address", billingAddressJSON)
             }
 
+            val bodyJSON = JSONObject()
             threeDSecureRequest?.also {
                 val verificationJSON = JSONObject()
                     .put("method", it.sca.name)
                 val attributesJSON = JSONObject()
                     .put("verification", verificationJSON)
                 cardJSON.put("attributes", attributesJSON)
+
                 // add return and cancel url when its supported
+                val applicationContextJSON = JSONObject()
```

Comment:
nit: `contextUrLJson` or `returnUrlJson` just because we have applicationContext meaning different thing in andorid

Reply:
That's a good call yeah. Went with `returnURLJSON` to differentiate 👍.